docs/releases/1.9.txt

```diff
@@ -167,6 +167,9 @@ Models
 * Added a system check to prevent defining both ``Meta.ordering`` and
   ``order_with_respect_to`` on the same model.
 
+* Added ``date`` lookup to :class:`~django.db.models.DateTimeField`, which
+  allows querying a ``DateTimeField`` by only the date portion.
```

Comment:
We should add something like "To use this feature, third-party database backends may need to implement a `datetime_cast_date_sql()` method." (might be better in a "Database backend API" section like we have in the 1.8 release notes).